src/textual/widgets/text_input.py

```diff
@@ -282,18 +281,6 @@ def on_key(self, event: events.Key) -> None:
             if visible_content_to_cursor_cell_len + key_cell_len >= available_width:
                 self.visible_range = start + key_cell_len, end + key_cell_len
             self._update_suggestion(event)
-        elif (
-            key == "ctrl+x"
-        ):  # TODO: This allows us to query and print the text input state
-            self.log(start=start)
-            self.log(end=end)
-            self.log(visible_content=visible_content)
-            self.log(visible_content_cell_len=visible_content_cell_len)
-            self.log(
-                visible_content_to_cursor_cell_len=visible_content_to_cursor_cell_len
-            )
-            self.log(available_width=available_width)
-            self.log(cursor_index=self._editor.cursor_index)
```

Comment:
I like the way that code tells a story... A dreadful story of a complex feature development.... :sweat_smile: